projects/Alarm clock/README.md

```diff
@@ -0,0 +1,15 @@
+<h1>Alarm Clock </h1>
+
+<h3> Import Required Library<h3>
+  <ul>
+<h4>from tkinter import *<h4>
+<h4>import datetime<h4>
+<h4>import time<h4>
+<h4>import winsound<h4>
+  <h4>from threading import *<h4></ul>
+    <br>
```

Comment:
readme.md file must contains steps for running your script no need to import code in readme.md